modules/es-cluster-health-is-red.adoc

```diff
@@ -0,0 +1,153 @@
++
+[source,terminal]
+----
+$ oc logs <elasticsearch_master_pod_name> -c elasticsearch -n openshift-logging
+----
+
+.. Review the logs of nodes that have not joined the cluster for issues by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc logs <elasticsearch_node_name> -c elasticsearch -n openshift-logging
+----
+
+. If all the nodes have joined the cluster, check if the cluster is in the process of recovering by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/recovery?active_only=true
+----
++
+If there is no command output, the recovery process might be delayed or stalled by pending tasks.
+
+. Check if there are pending tasks by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- health |grep  number_of_pending_tasks
+----
+
+. If there are pending tasks, monitor their status. If their status changes and indicates that the cluster is recovering, continue waiting. The recovery time varies according to the size of the cluster and other factors. Otherwise, if the status of the pending tasks does not change, this indicates that the recovery has stalled.
+
+. If it seems like the recovery has stalled, check if the `cluster.routing.allocation.enable` value is set to `none`, by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/settings?pretty
+----
+
+. If the `cluster.routing.allocation.enable` value is set to `none`, set it to `all`, by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cluster/settings?pretty -X PUT -d '{"persistent": {"cluster.routing.allocation.enable":"all"}}'
+----
+
+. Check if any indices are still red by running the following command and observing the output:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=_cat/indices?v
+----
+
+. If any indices are still red, try to clear them by performing the following steps.
+
+.. Clear the cache by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_cache/clear?pretty
+----
+
+.. Increase the max allocation retries by running the following command:
++
+[source,terminal]
+----
+$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query=<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.allocation.max_retries":10}'
```

Comment:
```suggestion
$ oc exec -n openshift-logging -c elasticsearch <elasticsearch_pod_name> -- es_util --query="<elasticsearch_index_name>/_settings?pretty -X PUT -d '{"index.allocation.max_retries":10}'"
```

Reply:
Per https://redhat-internal.slack.com/archives/C02321AKYTT/p1696866637850469?thread_ts=1696524477.252959&cid=C02321AKYTT we will leave this as is